doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -492,18 +507,10 @@ Response:
 
 ## Entity by ID [/v2/entities/{entityId}{?attrs,options}]
 
-### Retrieve entity [GET /v2/entities/{entityId}{?attrs}]
++ Parameters
```

Comment:
yes, and fixed as per our conversation f2f today

https://github.com/jmcanterafonseca/fiware-orion/commit/01c2427080c7f44b53a89e26ce38de3faf33d990